tripy/tripy/frontend/tensor.py

```diff
@@ -243,6 +261,25 @@ def eval(self) -> runtime.MemRefValue:
 
         return data
 
+    def _collect_storage_tensors(self):
```

Comment:
Shouldn't this be a property of the `Trace`?

Reply:
as in a static_method to call before making the `Trace` or pass the ops to trace and let it handle what the inputs and outputs are and then access `trace.inputs` and `trace_outputs` ?

Reply:
I think the latter could be a problem when user adds inputs too, since we would need to find a way to merge storage inputs and user passed inputs.

Reply:
No, I just mean the method should be defined under the Trace class instead of Tensor.

Reply:
okay done